integration-cli/docker_cli_rmi_test.go

```diff
@@ -13,27 +14,21 @@ func (s *DockerSuite) TestRmiWithContainerFails(c *check.C) {
 	errSubstr := "is using it"
 
 	// create a container
-	out, _, err := dockerCmdWithError("run", "-d", "busybox", "true")
-	if err != nil {
-		c.Fatalf("failed to create a container: %s, %v", out, err)
-	}
+	out, _ := dockerCmd(c, "run", "-d", "busybox", "true")
 
 	cleanedContainerID := strings.TrimSpace(out)
 
 	// try to delete the image
-	out, _, err = dockerCmdWithError("rmi", "busybox")
-	if err == nil {
-		c.Fatalf("Container %q is using image, should not be able to rmi: %q", cleanedContainerID, out)
-	}
-	if !strings.Contains(out, errSubstr) {
-		c.Fatalf("Container %q is using image, error message should contain %q: %v", cleanedContainerID, errSubstr, out)
-	}
+	out, _, err := dockerCmdWithError("rmi", "busybox")
+	// Container is using image, should not be able to rmi
+	c.Assert(err, checker.NotNil)
+	// Container is using image, error message should contain errSubstr
+	c.Assert(out, checker.Contains, errSubstr, check.Commentf("Container: %q", cleanedContainerID))
```

Comment:
As we have discussed in #16854, comment above `c.Assert` statement will be printed out together with `check.Commentf`, do you still see a need to put comment in `check.Commentf`?
@vdemeester